### PR TITLE
Swf configs

### PR DESCRIPTION
**Pull request by Jesse-Michael**

### CODE REVIEW

File: swf_workflow_config.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+func (s SimpleWorkflowConfig) GetDomain() string {
+	return *s.Domain
+}
+
+func (s SimpleWorkflowConfig) GetWorkflowType() WorkflowType {
+	return *s.WorkflowType
+}
+
```
Comment (bahern) [inline swf_workflow_config.go]: Shouldn't need getters/setters.  Just use the field on the struct.

File: swf_workflow_config.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	WorkflowType *WorkflowType `json:"workflow_type"`
+}
+
+func (s SimpleWorkflowConfig) GetDomain() string {
+	return *s.Domain
+}
+
+func (s SimpleWorkflowConfig) GetWorkflowType() WorkflowType {
+	return *s.WorkflowType
+}
```
Comment (bahern) [inline swf_workflow_config.go]: Shouldn't need getters/setters. Just use the field on the struct.

File: swf_client_config.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package remoteconfig
+
+type SimpleWorkflowClientConfig struct {
+	Region     *AWSRegion `json:"region,omitempty"`
+	Endpoint   *string    `json:"endpoint,omitempty" remoteconfig:"optional"`
+	DisableSSL *bool      `json:"disable_ssl,omit" remoteconfig:"optional"`
+}
+
+func (s SimpleWorkflowClientConfig) GetRegion() AWSRegion {
```
Comment (bahern) [inline swf_client_config.go]: Shouldn't need getters/setters. Just use the field on the struct.

**Comment by GeneticGenesis** [comment]
Looks like you're missing some dependencies in godep, namely github.com/aws/aws-sdk-go/service/swf

**Pull request closed without merging**